### PR TITLE
Merge Fan Art Contest results columns

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by agatemosu** [comment]
3965f1968ff61220fc7bf2534bdab07c6fdb09a7 applied suggested format by cl8n for tournaments
https://discord.com/channels/188630481301012481/218677502141399041/1167997675480821861

File: wiki/Contests/en.md [hunk 1]
```diff
@@ -158,40 +158,40 @@ At its launch, the contest was named "Best of", as the initial premise and expec
...
-| [#32 (15th Anniversary Art Contest)](https://osu.ppy.sh/home/news/2022-08-15-15-anniversary-art-contest) | osu! mascots exploring part of osu!'s history | ::{ flag=ID }:: [Dreamxiety](https://osu.ppy.sh/users/13103233) | ::{ flag=HK }:: [illuminae2005](https://osu.ppy.sh/users/29866964) | ::{ flag=GB }:: [urgutz](https://osu.ppy.sh/users/11731553) | [#1](https://osu.ppy.sh/home/news/2022-09-15-anniversary-art-contest-results) | [#1](https://osu.ppy.sh/community/contests/153) |
+| [#1](https://osu.ppy.sh/community/forums/topics/198551) | Pippi | ::{ flag=PH }:: [kunny](https://osu.ppy.sh/users/3931585) | ::{ flag=MY }:: [DumplingYumYum](https://osu.ppy.sh/users/1715930) | ::{ flag=HK }:: [Pokie](https://osu.ppy.sh/users/207340) | [Forum thread](https://osu.ppy.sh/community/forums/topics/210503) |
```
Comment (cl8n) [inline wiki/Contests/en.md:163]: to be very pedantic i think "forum topic" is the actual name for these

File: wiki/Contests/en.md [hunk 2]
```diff
@@ -158,40 +158,40 @@ At its launch, the contest was named "Best of", as the initial premise and expec
 
 The **Fan Art Contest** is a drawing contest where participating artists from the community are challenged to create the best fan art for osu! involving [osu!'s mascots](/wiki/Mascots) across various themes and art concepts.
 
-| Contest No. | Theme | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Results |
+| Contest No. | Theme | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Links |
```
Comment (cl8n) [inline wiki/Contests/en.md:161]: maybe "Results link" or something? this is still a column about results only.

**Review by cl8n — COMMENTED** [review]
seems fine to me but walavouchey said he doesn't want to change `#1`-links to this type of format /shrug

also i fixed a conflict on this branch so pls pull before more changes

File: wiki/Contests/en.md [hunk 3]
```diff
@@ -158,40 +158,40 @@ At its launch, the contest was named "Best of", as the initial premise and expec
...
+| [#9](https://osu.ppy.sh/home/news/2017-03-21-fanart-contest-9-spring) | Spring | ::{ flag=DE }:: [Evaine](https://osu.ppy.sh/users/7295733) | ::{ flag=JP }:: [suiseki](https://osu.ppy.sh/users/4792023) | ::{ flag=ID }:: [Karkovb152](https://osu.ppy.sh/users/5828215) | [News post](https://osu.ppy.sh/home/news/2017-04-27-spring-fanart-contest-results) |
+| [#10 (Summer 2017)](https://osu.ppy.sh/home/news/2017-06-15-mocha-fanart-contest-now-open) | Mocha In Summer | ::{ flag=FR }:: [Sylux6](https://osu.ppy.sh/users/2607507) | ::{ flag=PL }:: [sh00p](https://osu.ppy.sh/users/5272918) | ::{ flag=US }:: [Lilina](https://osu.ppy.sh/users/2920755) | [News post](https://osu.ppy.sh/home/news/2017-07-31-mocha-in-summer-fanart-contest-results) |
+| [#11 (Yuzu's New Look)](https://osu.ppy.sh/home/news/2017-08-21-yuzus-new-look-contest) | Yuzu's New Look | ::{ flag=AU }:: [ikaNyai](https://osu.ppy.sh/users/7088526) | ::{ flag=HK }:: [crystalsuicune](https://osu.ppy.sh/users/9974) | ::{ flag=US }:: [Thievley](https://osu.ppy.sh/users/4717672) | [News post](https://osu.ppy.sh/community/contests/49) |
```
Comment (cl8n) [inline wiki/Contests/en.md:173]: this one is a contest page, not news post

**Comment by TicClick** [comment]
closing due to the same reasoning as in https://github.com/ppy/osu-wiki/pull/10312

**Pull request closed without merging**